src/Illuminate/Notifications/Messages/SimpleMessage.php

```diff
@@ -21,6 +21,14 @@ class SimpleMessage
     public $subject;
 
     /**
+     * Greeting at the beginning of the notification.
+     * If null, a greeting depending on the level will be displayed.
```

Comment:
@GrahamCampbell I don't know what you mean, I added a new line, styleci is failing now